contracts/snapshots/MerkleSnapshot.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.11;
+
+import "../zeppelin//MerkleProof.sol";
+import "../ManagerProxyTarget.sol";
+
+
+contract MerkleSnapshot is ManagerProxyTarget{
+    mapping (bytes32 => bytes32) public snapshot;
+
+    constructor(address _controller) public Manager(_controller) {}
+
+    function setSnapshot(bytes32 _id, bytes32 _root) external onlyControllerOwner {
+        require(snapshot[_id] == bytes32(0), "Snapshot root for ID already exists");
```

Comment:
I wonder if it could make sense to not have this check here because:

- Not having this check here could make it easier to recover from the hopefully highly unlikely case where the initial snapshot is incorrect (i.e. an address is missing from the snapshot) such that it needs to be updated. This would be especially helpful if this contract is not upgradeable as a part of a proxy setup
- The Controller owner can still swap out snapshots if needed regardless of whether this check is here or not

Reply:
fixed in 2839cec